packages/jest-config/src/normalize.js

```diff
@@ -365,6 +365,18 @@ function normalize(config, argv) {
           ),
         );
         break;
+      case 'reporters':
+        if (!Array.isArray(config[key])) {
+          throw new Error(`
+            jest: "reporters" config must be an array.
+            Example:
+              "reporters": [
+                ['jest-cli/build/reporters/xunit', {"file": "report.xml"}]
```

Comment:
I'm pretty excited about this PR, I was looking for a way to customize the cobertura reporter's filename just yesterday. Would this new `reporters` key be a replacement for the existing `coverageReporters` config, or is this completely replacing some other "reporter"?

If these two config keys are meant to remain distinct, it sure would be nice to extend `coverageReporters` to accept the same optional per-reporter config:

```json
"jest": {
  "coverageReporters": [
    "text-summary",
    ["cobertura", { "file": "coverage.xml" }]
  ]
}
```

Oof, digging into the [reporter code](https://github.com/istanbuljs/istanbul-api/blob/master/lib/reporter.js#L59), it doesn't look like there's a way to pass reporter config through the instance API. Ah well, it was a nice idea.